src/main/java/com/openshift/internal/restclient/model/Pod.java

```diff
@@ -61,10 +76,35 @@ public String getHost() {
 		}
 		return images;
 	}
+	
+	/**
+	 * The logic of the method is a copied from 'podStatus' function of
+	 * [app/scripts/filters/resources.js] of [openshift/origin-web-console]
+	 */
 
 	@Override
 	public String getStatus() {
-		return asString(POD_STATUS);
+		if (has(POD_DELETION_TIMESTAMP)) {
+			return "Terminating";
```

Comment:
@adietish [it exists](https://github.com/openshift/origin-web-console/blob/7c57218bb83964ba0aab86ebf6ee42fb52aabf39/app/scripts/filters/resources.js#L1210). But again - not in the os-origin, but in os-web-console.